### PR TITLE
feat: add colleagues model,migration,seeder,factory

### PR DESCRIPTION
**Pull request by tektektek2000**

### CODE REVIEW

**Comment by TwoDCube** [comment]
The Travis failing is unrelated to this issue, it's tracked in VHD-31.

File: app/Http/Controllers/Admin/ColleaguesCrudController.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        ]);
+
+        $this->crud->addField([
+            'name'=>'image',
+            'label'=>'Profile image',
+            'type'=>'upload',
+            'upload'=>true,
+            'disk'=>'authors_images',
+        ]);
+
```
Comment (Sasszem) [inline app/Http/Controllers/Admin/ColleaguesCrudController.php]: "Awards" field missing

File: app/Http/Controllers/Admin/ColleaguesCrudController.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            'name'=>'roles',
+            'label'=>'Roles',
+            'type'=>'textarea',
+        ]);
+        
+        $this->crud->addField([
+            'name'=>'category',
+            'label'=>'Category',
+            'type' => 'select_from_array',
+            'options' => [0 => 'Brass', 1 => 'Teachers', 2=>'lecturer', 3=>'Administrator', 4=>'Kitchen Staff', 5=>'Maintenance Staff'],
```
Comment (Sasszem) [inline app/Http/Controllers/Admin/ColleaguesCrudController.php]:
```suggestion
            'options' => [0 => 'Brass', 1 => 'Teachers', 2=>'Lecturer', 3=>'Administrator', 4=>'Kitchen Staff', 5=>'Maintenance Staff'],
```

File: app/Http/Controllers/Admin/ColleaguesCrudController.php [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            'allows_null' => false,
+        ]);
+        
+        $this->crud->addField([
+            'name'=>'image',
+            'label'=>'Profile image',
+            'type'=>'image',
+            'prefix'=>'storage/authors_images/'
```
Comment (Sasszem) [inline app/Http/Controllers/Admin/ColleaguesCrudController.php]: This full field is bad:
 - Your not uploading the file to anywhere
 - The prefix you `set` points to the wrong disk
 - The prefix does not need a '/' at the end, in fact it makes the resulting URL invalid

File: app/Models/Colleagues.php [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+class Colleagues extends Model
+{
+    use CrudTrait;
+    protected $table = 'colleagues_data';
+    public $timestamps = false;
+    protected $fillable = ['name','jobs','subjects','roles','awards','image','category'];
+    protected $hidden = [];
+    //Follows the format of the old website. For example jobs=igazgató|subjects=matematika,technika
+    //|roles:"A Verseghy Diákokért" Alapítvány kuratóriumának elnöke,|awards=Graphisoft-díj (2002.)
```
Comment (Sasszem) [inline app/Models/Colleagues.php]: You should add a mutator to upload the image, see https://github.com/Laravel-Backpack/CRUD/issues/1219#issue-293761603 for an example

File: app/Http/Controllers/Admin/ColleaguesCrudController.php (hunk 1 above)
Comment (Sasszem) [reply app/Http/Controllers/Admin/ColleaguesCrudController.php]: Added this field.

File: app/Http/Controllers/Admin/ColleaguesCrudController.php (hunk 2 above)
Comment (Sasszem) [reply app/Http/Controllers/Admin/ColleaguesCrudController.php]: Fixed.

File: app/Models/Colleagues.php (hunk 4 above)
Comment (Sasszem) [reply app/Models/Colleagues.php]: Added mutator.

File: app/Http/Controllers/Admin/ColleaguesCrudController.php (hunk 3 above)
Comment (Sasszem) [reply app/Http/Controllers/Admin/ColleaguesCrudController.php]: Fixed uploading.

**Pull request merged**